File: Container_loop/4.py
in_str = input("입력해주세요.\n")
members = ["egoing", "k88"]

# i = 0
#
# for member in members:
#     if in_str == member:
#         print("hello" + in_str)
#         i = i + 1
#
# if i == 0:
#     print("who are you?")


for member in members:
    if in_str == member:
        print("hello" + in_str)
        import sys
        sys.exit()

print("who are you?")

File: Container_loop/4.rb
puts("아이디를 입력하세요\n")
in_str = gets.chomp()

# real_egoing = "11"
# real_k88 = "ab"
#
# if real_egoing == in_str
#   puts("Hello!, egoing")
# elsif real_k88 == in_str
#   puts("Hello!, k88")
# else
#   puts("Who are you?")
# end

members = ["egoing", "k88"]

for member in members do
  if member == in_str
    puts("Hello " + member)
    exit
  end
end

puts("who are you?")
